packages/core/src/sdk/useStore.ts

```diff
@@ -1,3 +1,5 @@
+'use client'
```

Comment:
**(non-blocking)**: Why we need use client here?

Reply:
`use store` is used in `useSearchHistory` and I was getting an error in that file. As this hook will always be a client component, I don't see a problem in adding a `use client` in it